### PR TITLE
add link & fix typo

### PR DESCRIPTION
**Pull request by Osier-Yi**

### CODE REVIEW

File: _docs/19-privacy-attacks.md [hunk 1]
```diff
@@ -349,7 +362,7 @@ def hook_on_data_injection_sav_data(ctx):
 
 
 ## 4. Contribute Your Attacker to FederatedScope
-Users are welcome to contribute their own attack methods to FederatedScope. Please refer LINK_TO_ADD for more details. 
+Users are welcome to contribute their own attack methods to FederatedScope. Please refer [Contributing to FederatedScope]({{"/docs/contributor/"}}) for more details.
```
Comment (joneswong) [inline _docs/19-privacy-attacks.md:365]: please change to `{{ "/docs/contributor/" | relative_url }}`

File: _docs/19-privacy-attacks.md [hunk 2]
```diff
@@ -258,18 +271,18 @@ The following is an example in property inference attack where the attacker (ser
             print(self.pia_results)
 ```
 
-### 3.2 Client as the Attacker
-When the attacker is one of the client:
+### 3.3 Client as the attacker
+When the attacker is one of the clients:
 * If the attack actions only happen in the local training procedure, users only need to define the wrapping function to wrap the trainer and add the attack actions. 
-* If the atttack actions also happen in the end of the FL training, users need to overload the client class and modify its`callback_funcs_for_finish` function. 
+* If the attack actions also happen at the end of the FL training, users need to overload the client class and modify its'callback_funcs_for_finish` function.
```
Comment (joneswong) [inline _docs/19-privacy-attacks.md:277]: the syntax seems inconsistent

**Review by joneswong — CHANGES_REQUESTED** [review]
Please fix the issues. Thanks!

File: _docs/19-privacy-attacks.md (hunk 1 above)
Comment (Osier-Yi) [reply _docs/19-privacy-attacks.md]: fix~

File: _docs/19-privacy-attacks.md (hunk 2 above)
Comment (Osier-Yi) [reply _docs/19-privacy-attacks.md]: fix

File: _docs/19-privacy-attacks.md [hunk 3]
```diff
@@ -45,7 +45,7 @@ According to the attacker's actions, the privacy attacks can be divided into pas
 
 ## 2. Usage of Attack Module
 ### 2.1 Preliminary
-Before calling the attack module, the user should make sure that the FL has been set up. For more details of setting up FL, please refer [quick start]({{"/docs/quick-start/"}}) and [start your own case]({{"/docs/own-case/"}}). 
+Before calling the attack module, the user should make sure that the FL has been set up. For more details of setting up FL, please refer [quick start]({{"/docs/quick-start/" | relative_url }}) and [start your own case]({{"/docs/own-case/" | relative_url }}).
```
Comment (FederatedScopeTeam) [inline _docs/19-privacy-attacks.md:48]: It is necessary to put a space in front of permalink, i.e., `{{ "/doc/quick-start/" | relative_url }}`

**Review by FederatedScopeTeam — CHANGES_REQUESTED** [review]
See my inline comment. thx!

File: _docs/19-privacy-attacks.md (hunk 3 above)
Comment (Osier-Yi) [reply _docs/19-privacy-attacks.md]: > It is necessary to put a space in front of permalink, i.e., `{{ "/doc/quick-start/" | relative_url }}`

fix~

**Review by FederatedScopeTeam — APPROVED** [review]
Approve!

**Review by joneswong — APPROVED** [review]
LGTM

**Pull request merged**